lively.ide/value-widgets.js

```diff
@@ -320,6 +320,7 @@ export class NumberWidget extends Morph {
   }
 
   update (v, fromScrubber = true) {
+    if (this.inactive) return;
```

Comment:
Where is this set?

Reply:
Good catch, that was a half-baked idea that was not necessary in the end. Removed!